test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -369,7 +369,7 @@
                           {:uri "/v1/certificate_statuses/"
                            :request-method :get})]
             (is (= 400 (:status response)))
-            (is (= "text/plain" (get-in response [:headers "Content-Type"])))
+            (is (re-matches #"text/plain.*" (get-in response [:headers "Content-Type"])))
```

Comment:
per @camlow325 it would be good to just assert that this = `text/plain; charset=utf-8`